src/store/FirestoreInterfaces.tsx

```diff
@@ -1,12 +1,15 @@
+export interface SiteCollection {
+  [key: string]: SiteObject;
+}
 export interface SiteObject {
   address: string;
+  equipmentUnits: EquipmentUnit[];
```

Comment:
I don't know if this will cause issues in the future or not, but equipmentUnits is a map in firestore, with keys being equipment name's.